Tests/UnitTests/Android/app/src/main/cpp/CMakeLists.txt

```diff
@@ -8,14 +8,17 @@ project(UnitTestsJNI)
 get_filename_component(UNIT_TESTS_DIR "${CMAKE_CURRENT_LIST_DIR}/../../../../.." ABSOLUTE)
 get_filename_component(REPO_ROOT_DIR "${UNIT_TESTS_DIR}/../.." ABSOLUTE)
 
+include("${UNIT_TESTS_DIR}/../../content.cmake")
+npm(install "${UNIT_TESTS_DIR}/../" "UnitTests")
```

Comment:
I don't like this. It would be better if we didn't pollute the source dir and instead do it in the CMake binary dir.